### DISCUSSION TITLE
How to control string quoting on a per-property basis?

### DISCUSSION THREAD

**Discussion by robross0606** [eemeli/yaml#453] [Q&A] [answered]
We need to update two different YAM properties.  For one, we would like to have it single-quoted when serialized.  For the other, we would like it to not have quotes when serialized.  Is it possible with this library to specify how a single property is serialized on stringify?

**Comment by robross0606** [comment, marked as answer]
I _think_ I see how to do this now:
```js
const doc = new Document()

const firstProp = doc.createNode("hello")
firstProp.type = 'PLAIN'
doc.set('first', firstProp)

const secondProp= doc.createNode("world")
secondProp.type = 'QUOTE_DOUBLE'
doc.set('second', secondProp)

doc.toString()
```

Is that correct?

**Reply by eemeli** [reply]
Yes, that looks like it should work.
